package io.github.loggerworld.exception

class UserAlreadyExistsException(message: String) : RuntimeException(message)